packages/forms-web-app/src/controllers/examination/personal-information-which/utils/savePersonalInformationFlags.js

```diff
@@ -0,0 +1,35 @@
+const { makeIntoArray } = require('../../select-file/utils/helpers');
```

Comment:
Shall we move this file into session under - personal info ?

Reply:
I think it would make sense where it currently is as it is specific to this page.